### PR TITLE
issue_10/contribute_page

### LINKED ISSUE #10: Create the contribution page
Create a page with basic information about how another developers could contribute with TryTechLabs projects, as showed on the design reference bellow.

**:warning: Important to observe**

1. The Github button should redirect to the Organization page on GitHub, so it could be a static link.
2. The footer will be created on another issue.

Contribute

### PR DESCRIPTION
**Pull request by claytonrv**
Create the Contribute page structure, components and styles, as defined on the design, as showed bellow.
Contribute

### CODE REVIEW

**Comment by claytonrv** [comment]
Sorry by this huge PR, but I found a bunch of issues on the project (style, usability and the tests that was on the wrong "pattern" [out of src/tests] - what forces you to define the complete path of the tests to run it). 
On this way, the css grid pattern of many pages was created on a weird way (each page has his own grid style, basically). All the pages are following the same grid principles now, only with small particularities. There's minor changes on the font style and size on all pages.

The result of the Contribute page is the following:
Screenshot_20201229_184041

I've made some changes on the home screen layout too, once it was on a different grid pattern, comparing with the other pages. Now it's following the same grid css pattern of the other pages (what makes the page looks better too):
Before:
Screenshot_20201202_220337

After:
Screenshot_20201229_232139

The changes on the home screen are more visible when you inspect it or when you resize the screen. Like it's now, it will be the same page independent of the size of the screen (not applied to the mobile sizes - need to work on it separately)

File: src/pages/Landing.tsx [hunk 1]
```diff
@@ -14,25 +14,18 @@ function Landing() {
...
-                                    <div className="green-dot"></div>
-                                </div>
-                            </div>
-                            <p>We're a group of friends trying to do cool things.</p>
-                        </div>
-                        <div className="grid_column">
-                            <img src={Developer} alt="Man in front of a laptop" className="home-developer" />
-                        </div>
+                <div className="main-page-logo">
+                    <SiteLogo className="" width={277} height={166} />
```
Comment (mspdl) [inline src/pages/Landing.tsx:18]: The only change I'd recomend is to move width and height for the css file creating a new className or just remove the className

**Review by mspdl — APPROVED** [review]
I don't know much about tests on React, but I read the code and I didn't find anything wrong. The new components and css organization are excellent. I added just a small tip on Landing component, but it's ok if you don't accept.

File: src/pages/Landing.tsx (hunk 1 above)
Comment (claytonrv) [reply src/pages/Landing.tsx:18]: I could @mspdl, but the idea on that is to reuse the component. If you take a look on `Landing.tsx` and `NavigationMenu.tsx` you'll see that I'm using this SiteLogo component on both, changing only it's size (`width` and `height`). That's why I've created it outside a css class. 🙂

**Review by lucasmauro — APPROVED** [review]
To the moon

**Pull request merged**